v2/apiserver/internal/core/jobs_test.go

```diff
@@ -250,6 +250,113 @@ func TestJobsServiceUpdateStatus(t *testing.T) {
 	}
 }
 
+func TestJobsServiceCleanup(t *testing.T) {
```

Comment:
Should we add a success case here? (or intended to be covered by the corresponding sdk test?)

Reply:
Oversight. Nice catch. Fixing.

Reply:
Fixed.